pkgs/by-name/li/libmamba/package.nix

```diff
@@ -0,0 +1,44 @@
+{ fetchFromGitHub
+, lib
+, stdenv
+, cmake
+, fmt
+, spdlog
+, tl-expected
+, nlohmann_json
+, yaml-cpp
+, simdjson
+, reproc
+, libsolv
+, curl
+, libarchive
+, zstd
+, bzip2
+, python3
+, ...
+}:
+let
+  mamba-libsolv = libsolv.override { withConda = true; };
+in
```

Comment:
I guess this is arguable, but I'd rather add a top-level attribute for this (e.g. `libsolv-with-conda = ...` in `all-packages.nix`) and take it as is from callPackage.